.goreleaser.yml

```diff
@@ -34,6 +34,19 @@ archives:
     amd64: x86_64
 checksum:
   name_template: 'checksums.txt'
+signs:
+  - cmd: cosign
+    env:
+      - COSIGN_EXPERIMENTAL=1
+    signature: "${artifact}.sig"
+    certificate: "${artifact}.pem"
+    args:
+      - sign-blob
+      - '--oidc-issuer={{if index .Env "CI"}}https://token.actions.githubusercontent.com{{else}}https://oauth2.sigstore.dev/auth{{end}}'
```

Comment:
Is the `CI` env var the best way we have to detect we're running in CI? Maybe the presence of the `GITHUB_SHA` env var?

(edit: Oh, this is copied from goreleaser/goreleaser's similar config. Does the goreleaser action set the `CI` env var? 🤔)

Reply:
I think this approach might help for local development, so, to ensure that someone who might want to run the goreleaser commands locally still can do that If I'm not mistaken.

Reply:
nvm, GitHub Actions says `CI` is always set during GHA workflow execution, so this is right.